dandelion-voting/app/src/hooks/useExtendedVoteData.js

```diff
@@ -52,5 +55,6 @@ export default function useExtendedVoteData(vote) {
     userBalanceNowPromise,
     canUserVotePromise,
     canExecutePromise,
+    startTimestamp,
```

Comment:
Why are we getting the startTimeStamp inside the extendedVoteData instead of inside each voteData?

Reply:
The `startTimeStamp` is a function that uses a promise to fetch and return the timestamp for the `startBlock`. As this is used only when the `voteDetails` is rendered, i think it makes sense to load it dynamically when requested rather than fetching it for all the votes in `app-logic`

Reply:
Makes sense